{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -47,14 +47,15 @@ classifiers = [
 {%- if cookiecutter.backend == "hatch" %}
 dynamic = ["version"]
 {%- endif %}
-dependencies = ["pybamm"]
+dependencies = ["pybamm", "cookiecutter"]
```

Comment:
Is the cookiecutter dependency required for the template? Where is it being used?

Reply:
It is not required imo.